skycoin/skywire
# Parity between UI and CLI

Did you run `make format && make check`? Yes

Fixes #920

 Changes:	
- Add `summary` command as `skywire-cli visor summary`
- Add `update` command as `skywire-cli visor update`
- Add `BUILDTAG` to skywire-cli on goreleaser.

**How to test this PR**:
- **summary**: 
  - Run `skywire-cli visor summary`.
- **update**: 
  - Build with skybian *BUILDTAG* flag `make build BUILDTAG=skybian`.
  - Run `skywire-cli visor update` to update visor.
  - (Because of no *BUILDTAG* set on skywire-cli v0.5.1) Build with skybian *BULDTAG* again.
  - Run `skywire-cli visor update` to get message about visor has the latest version.
  - For get error on non-skybian *BUILDTAG*, make build without flag and run update command. Now should get unable to update because of non-skybian *BUILDTAG*.
- **config**
  - Multiple flags added to change their values on config file. Listed as bellow:
    - `vpn-client-killswitch`: could be *true* or *false*.
    - `add-vpn-client-server`: add vpn server address to vpn-client.
    - `add-vpn-client-passcode`: add passcode of vpn server to vpn-client.
    - `reset-vpn-client`: reset all config of vpn-client.
    - `add-vpn-server-passcode`: add passcode for vpn-server.
    - `vpn-server-secure`: could be *true* or *false*.
    - `reset-vpn-server`: reset all config of vpn-server.
    - `add-skysocks-client-server`: add skysocks server pk to skysocks-client.
    - `reset-skysocks-client`: reset skysocks-client config.
    - `add-skysocks-passcode`: set passcode for skysocks server.
    - `reset-skysocks`: reset skysocks server config.
    - `set-public-autoconnect`: could be *true* or *false*.
    - `set-minhop`: should be an integer number greater equal 0.
